### PR TITLE
Add support for Terraform v0.12

### PR DESCRIPTION
**Pull request by kopachevsky**
Issue #21

### CODE REVIEW

File: test/ci_integration.sh [hunk 1]
```diff
@@ -42,6 +42,7 @@ setup_environment() {
   export TF_VAR_parent_resource_folder="$PARENT_RESOURCE_FOLDER"
   export TF_VAR_parent_resource_billing_account="$PARENT_RESOURCE_BILLING_ACCOUNT"
   export TF_VAR_parent_resource_organization="$PARENT_RESOURCE_ORGANIZATION"
+  export TF_LOG=DEBUG
```
Comment (aaron-lane) [inline test/ci_integration.sh]: We shouldn't need debug level logs for regular CI runs.

```suggestion
```

File: test/fixtures/computed_values/outputs.tf [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/**
+/**pubsub-project-local
```
Comment (aaron-lane) [inline test/fixtures/computed_values/outputs.tf]: This appears to have been mistakenly added.

```suggestion
/**
```

File: test/integration/bigquery/organization/controls/gcp.rb [hunk 3]
```diff
@@ -18,18 +18,11 @@
 control "gcp" do
   title "Log exports - organization level bigquery destination - native resources"
 
-  describe google_bigquery_dataset(
-    project: destination_map["project"],
-    name: destination_map["resource_name"],
-  ) do
-    it { should exist }
-  end
+  describe "google_bigquery_dataset" do
```
Comment (aaron-lane) [inline test/integration/bigquery/organization/controls/gcp.rb]: What is the motivation for this change?

File: .kitchen.yml [hunk 4]
```diff
@@ -77,10 +77,6 @@ suites:
       root_module_directory: test/fixtures/pubsub/project/
     verifier:
       systems:
-        - name: inspec-gcp
```
[the remark this answers is not in the document]
Comment (kopachevsky) [reply .kitchen.yml]: it was not worked, but commit itself is exident, I'll revert this removal

File: Gemfile [hunk 5]
```diff
@@ -12,8 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ruby '2.5.3'
+ruby '2.6.3'
 
 source 'https://rubygems.org/' do
   gem 'kitchen-terraform', '~> 4.0'
+  gem 'sourcify'
```
[the remark this answers is not in the document]
Comment (kopachevsky) [reply Gemfile]: it was for debugging, removing

File: main.tf [hunk 6]
```diff
@@ -18,65 +18,66 @@
...
+  log_sink_resource_name   = local.is_project_level ? element(concat(google_logging_project_sink.sink.*.name, list("")), 0) : local.is_folder_level ? element(concat(google_logging_folder_sink.sink.*.name, list("")), 0) : local.is_org_level ? element(concat(google_logging_organization_sink.sink.*.name, list("")), 0) : local.is_billing_level ? element(concat(google_logging_billing_account_sink.sink.*.name, list("")), 0) : ""
+  log_sink_parent_id       = local.is_project_level ? element(concat(google_logging_project_sink.sink.*.project, list("")), 0) : local.is_folder_level ? element(concat(google_logging_folder_sink.sink.*.folder, list("")), 0) : local.is_org_level ? element(concat(google_logging_organization_sink.sink.*.org_id, list("")), 0) : local.is_billing_level ? element(concat(google_logging_billing_account_sink.sink.*.billing_account, list("")), 0) : ""
 }
 
 #---------------------#
 # Variable validation #
 #---------------------#
+/*
```
[the remark this answers is not in the document]
Comment (kopachevsky) [reply main.tf]: yes, it explains explicitly in readme docs

File: test/integration/bigquery/organization/controls/gcp.rb (hunk 3 above)
Comment (kopachevsky) [reply test/integration/bigquery/organization/controls/gcp.rb]: debug leftover as well

**Comment by kopachevsky** [comment]
@aaron-lane resolved all comments, forgot to clean up debug mess when creating this PR, can you please review again
